src/app/services/embed.service.ts

```diff
@@ -0,0 +1,23 @@
+import { Project } from 'src/app/models/domain/project';
+import { Injectable } from '@angular/core';
+import { HttpClient } from "@angular/common/http";
+import { API_CONFIG } from "../config/api-config";
+import { HttpBaseService } from "./http-base.service";
+import { EmbeddedProject } from './../models/domain/embedded-project';
+
+import { EmbeddedProjectResource } from './../models/resources/embedded-project-resource';
+import { Observable } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class EmbedService extends HttpBaseService<EmbeddedProject,EmbeddedProjectResource,null> {
```

Comment:
Giving null to define the type of the TUpdate generic seems strange. This seems like it could cause bugs later down the line. In case we don't need a Update I suggest implementing a alternative HttpBaseService which has no Update implementation. Alternatively if we need the Update functionality a Resource should be created and added here